src/handlers.cpp

```diff
@@ -623,6 +623,8 @@ void ImpiRegistrationStatusTask::run()
       _visited_network = _dest_realm;
     }
     _authorization_type = _req.param("auth-type");
+    std::string sos = _req.param("sos");
+    _emergency = sos == "true" ? true : false;
     TRC_DEBUG("Parsed HTTP request: private ID %s, public ID %s, visited network %s, authorization type %s",
```

Comment:
Should "sos" and "true" be in a header, as they are just strings?
Also, since these are passed from sprout, shouldn't they be in a shared header so that we don't change them in one and break everything?

Although, the "auth-type" just above them is not, so if strings are used in the rest of this file, I'd be happy for you to leave these new strings you've added.